src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -992,10 +992,14 @@ private Map<String, BigDecimal> getInactiveJsonMap(RoundTally roundTally) {
     Map<String, BigDecimal> inactiveMap = new HashMap<>();
     Pair<String, StatusForRound>[] statusesToPrint =
         new Pair[] {
-          new Pair<>("overvotes", StatusForRound.INACTIVE_BY_OVERVOTE),
-          new Pair<>("skippedRankings", StatusForRound.INACTIVE_BY_SKIPPED_RANKING),
-          new Pair<>("exhaustedChoices", StatusForRound.INACTIVE_BY_EXHAUSTED_CHOICES),
-          new Pair<>("repeatedRankings", StatusForRound.INACTIVE_BY_REPEATED_RANKING)
+          new Pair<>("overvotes",
+                     StatusForRound.INVALIDATED_BY_OVERVOTE),
+          new Pair<>("skippedRankings",
+                     StatusForRound.INVALIDATED_BY_SKIPPED_RANKING),
+          new Pair<>("repeatedRankings",
+                     StatusForRound.INVALIDATED_BY_REPEATED_RANKING),
+          new Pair<>("exhaustedChoices",
+                     StatusForRound.EXHAUSTED_CHOICE),
```

Comment:
exhausted is extra here? Why was that not here before? It looks like this list is used to drive the properties of the summary.json. It looks like `summary.json` had this property before, just always 0's. Maybe this just provides the correct lookup for the ResultsWriter to fill it in? https://github.com/BrightSpots/rcv/pull/857/files#diff-581edbf64dc2eb31f841777e3cfdc5e462066e8cd7de90c026e68538b8b8a46cL94-R94

Reply:
oops! it is just a different order